go.mod

```diff
@@ -202,7 +210,7 @@ require (
 	go.uber.org/atomic v1.11.0
 	go.uber.org/automaxprocs v1.5.3
 	go.uber.org/dig v1.17.0
-	go.uber.org/fx v1.18.2
+	go.uber.org/fx v1.20.0
```

Comment:
@hush-hush I think there were issues when updating `fx` at some point, was it just due to the version update or was it the other changes ? https://github.com/DataDog/datadog-agent/pull/17783
(in doubt we could just set the version in `pkg/util/fxutil/go.mod` to 1.18.2 to avoid the change)